Full command path headers: add note regarding md extensions order (#88)
mkdocs/mkdocs-click: README.md

    - mkdocs-click
```

> Note: mind the order, `attr_list` should be above `mkdocs-click`, otherwise the feature does not work.

Reviewer: ```suggestion
***IMPORTANT:*** `attr_list` must come before `mkdocs-click` for this to work.
```
